outscale/data_source_outscale_load_balancer_test.go

```diff
@@ -15,6 +15,7 @@ func TestAccOthers_LBU_basic(t *testing.T) {
 	var conf oscgo.LoadBalancer
 
 	zone := fmt.Sprintf("%sa", utils.GetRegion())
+	number := utils.RandIntRange(20, 35)
```

Comment:
can you add a const, for 20 and 35. 
so instead of `number := utils.RandIntRange(20, 35)` you have `number := utils.RandIntRange(MAX_TEST_MACHIN, MAX_TEST_MACHIN2)`